README.md

```diff
@@ -1,9 +1,82 @@
 # MILLION DOLLAR DREAM
 Every man has a price.
 
-This creates a bloom filter to store and lookup file hashes in a space efficent
+This project assumes you have a whole boat load (10k+?) of files you have calculated
+a baseline of and that you want to compare to see if any have changed.  This 
+creates a bloom filter to store and lookup file hashes in a space efficient
 manner. 
 
+## OPTIONS
+
+```bash
+python3 ./million_dollar_dream.py 
+usage: ./million_dollar_dream.py <calculate|lookup|fromfile|filters> <filterfile> <file1> [file2 ...]
+```
+
+* calculate - given an input of file(s), calculate all the hashes and store them in the filter file
+* lookup - tbd
+* fromfile - tbd
+* filters - given an input of file(s), see if any of them have changed based on the  the filter 

```

Comment:
This description is for `lookup` rather than `filters`.  The `filters` command is a simple package manager for filter files.

By default, the repo URL in `config.json` is set to my repo at https://github.com/roberson-io/mdd_filters/raw/master/repo/. Until I fix something related to the last_updated timestamp, I'd recommend using https://github.com/droberson/mdd_filters/raw/master/repo/.

If you don't have a `config.json` file, just try running one of the filter commands to generate one like:
`./million_dollar_dream.py filters list`

`filters` commands:
`./million_dollar_dream.py filters fetch <filter_name>`
Fetch a filter file from the repo set in `config.json`.
For example, `./million_dollar_dream.py filters fetch drupal`

`./million_dollar_dream.py filters list`
List the locally installed filters.

`./million_dollar_dream.py filters list remote`
List the filters available at the repo set in `config.json`.

`./million_dollar_dream.py filters list <url>`
List the filters available at the URL provided.

`./million_dollar_dream.py filters update`
If any of the locally installed filters have a newer version at the repo set in `config.json`, fetch them.